src/test/java/org/purescript/psi/PSImportDeclarationImplTest.kt

```diff
@@ -121,4 +121,94 @@ class PSImportDeclarationImplTest : BasePlatformTestCase() {
         assertContainsElements(fuz.namedImports, "hiding")
     }
 
-}
\ No newline at end of file
+    fun `test parses import declaration children`() {
+        val file = myFixture.configureByText(
+            "Foo.purs",
+            """
+                module Foo where
+                import Adam
+                import Adam.Nested
+                import Adam.Aliased as A
+                import Bertil ()
+                import Bertil.Nested ()
+                import Bertil.Aliased () as B
+                import Caesar hiding ()
+                import Caesar.Nested hiding ()
+                import Caesar.Aliased hiding () as C
+            """.trimIndent()
+        ) as PSFile
```

Comment:
I'm not entirely sure of this test. Maybe it's excessive to test all these combinations. I'd happily split it or change it if you feel like it'd be better.

Reply:
I'm not sure `()` should be is valid?

```
module Tmp where

import Prelude ()
```

```
at Tmp.purs:3:17 - 3:18 (line 3, column 17 - line 3, column 18)

  Unable to parse module:
  Unexpected token ')'
```

Reply:
You're right, I think I confused it with some other thing that allowed empty parenthesis. I'll change it in some upcoming PR since this one is merged.